docs/introduction/QuickStart.mdx

```diff
@@ -0,0 +1,337 @@
+take whatever value it returns, and save that value inside as the current "state". We can then ask the store what its current state value is.
+
+```js {11-15}
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { configureStore } from '@reduxjs/toolkit'
+
+import App from './App'
+
+function rootReducer(state, action) {
+  return 'Hello Redux!'
+}
+
+const store = configureStore({
+  reducer: rootReducer
+})
+
+console.log(store.getState())
+
+const rootElement = document.getElementById('root')
+ReactDOM.render(<App />, rootElement)
+```
+
+If you look at the console in your browser's DevTools (or in CodeSandbox), you should see that the words `"Hello Redux!"` were printed.
+
+### Set Up the Provider Component
+
+We've got a Redux store with its state inside, but now we need to find a way to use that state inside our `<App>` component.
+
+It's _possible_ that we could move the store into a separate file, import that store into the `App.js` file, and call `store.getState()`. But, as the
+app gets bigger, we'd have to import the store into _every_ file. That's not a good idea.
+
+This is where the React-Redux library comes in. It gives us a way to read data from the store in any component in our application, without needing
+to import the store into every file.
+
+To do this, we need to pass the store into a React-Redux `<Provider>` component.
+
+<Note type="success">
+
+#### Key Concept: `<Provider>`
+
+The React-Redux library has a `<Provider>` component. It uses [React's context API](https://reactjs.org/docs/context.html) to pass the Redux store
+to all components inside of itself.
+
+Every React-Redux app needs to put a `<Provider>` component around the main `<App>` component so that the whole app can use the store.
+
+</Note>
+
+We'll import the `Provider` component type from React-Redux, render a `<Provider>` around our `<App>`, and then pass the store to the `<Provider>`. (We'll also remove the console log, since we don't need that any more.)
+
+```js {4,18-25}
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { configureStore } from '@reduxjs/toolkit'
+import { Provider } from 'react-redux'
+
+import App from './App'
+
+function rootReducer(state, action) {
+  return 'Hello Redux!'
+}
+
+const store = configureStore({
+  reducer: rootReducer
+})
+
+const rootElement = document.getElementById('root')
+
+ReactDOM.render(
+  <Provider store={store}>
+    <App />
+  </Provider>,
+  rootElement
+)
+```
+
+### Show the State in the App Component
+
+The last step for the initial setup is to update our `<App>` component to read the state value from the Redux store.
+
+We've already seen that the React-Redux `<Provider>` component makes the store available to any components inside. To go along with that, React-Redux
+includes a React hook called `useSelector`, which React function components can call to retrieve the state values from the store.
```

Comment:
We should link to the React hooks API docs here. Might also be worth mentioning that React hooks is a pre-req for this page, since the end user could be new to the React hooks API as well.